lib/queue/producer.ts

```diff
@@ -157,151 +227,6 @@ export class Producer implements QueueProducer {
 		}
 	}
 
-	// FIXME this function exists solely for the purpose of allowing upstream code
-	// to put stuff "in the queue" ( = the request table on db) and call worker.execute
-	// right after. Fix upstream code by calling queue.enqueue and let the worker deal
-	// with the request asynchronously. Once done, turn this function private or merge
-	// it with `enqueue`
-	async storeRequest(
-		actor: string,
-		session: string,
-		options: ProducerOptions,
-	): Promise<ActionRequestContract> {
-		const id = uuidv4();
-		const slug = `action-request-${id}`;
-
-		logger.debug(options.logContext, 'Storing request', {
-			actor,
-			request: {
-				slug,
-				action: options.action,
-				card: options.card,
-			},
-		});
-
-		// Use the request session to retrieve the various contracts, this ensures that
-		// the action cannot be run if the session doesn't have access to the contracts.
```

Comment:
Should probably keep these access checks somewhere in the worker, running them before inserting `action-request` contracts.